tests/test_routes.py

```diff
@@ -1024,6 +1024,27 @@ def test_usn(self):
         assert response.status_code == 200
         assert response.json["cves_ids"] == self.models["notice"].cves_ids
 
+    def test_multiple_usn(self):
+        response = self.client.get("/security/notices.json")
+
+        assert response.status_code == 200
+        # Should include cve
+        assert (
+            self.models["notice"].id
+            in response.json["notices"][0]["cves"][0]["notices_ids"]
+        )
+
+    def test_page_notice(self):
```

Comment:
Can we test the full functionality here as well? I think we should at least assert that querying by cve-id, details, and releases works, WDYT?

Reply:
Makes sense to me! Will add the tests